guides/source/active_record_callbacks.md

```diff
@@ -211,12 +487,15 @@ You have initialized an object!
 => #<User id: 1>
 ```
 
-[`after_find`]: https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-after_find
-[`after_initialize`]: https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-after_initialize
+[`after_find`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-after_find
+[`after_initialize`]:
+    https://api.rubyonrails.org/classes/ActiveRecord/Callbacks/ClassMethods.html#method-i-after_initialize
 
 ### `after_touch`
 
-The [`after_touch`][] callback will be called whenever an Active Record object is touched.
+The [`after_touch`][] callback will be called whenever an Active Record object
+is touched.
```

Comment:
I felt like I was wondering what is the meaning of "touching" a bit when I got to this section... not sure if we need to explain it further here, but we could at least point to the `touch` API Documentation for more info?

Reply:
I've added this - NOTE: You can read more about `touch` [here](https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-touch)